package/network/services/dropbear/files/dropbear.init

```diff
@@ -177,7 +177,7 @@ validate_section_dropbear()
 		'SSHKeepAlive:uinteger:300' \
 		'IdleTimeout:uinteger:0' \
 		'MaxAuthTries:uinteger:3' \
-		'RecvWindowSize:uinteger:262144' \
+		'RecvWindowSize:uinteger:0' \
```

Comment:
Why do you change this back? It was set in 865ae1c10c65001813413da95eb1b8cd06a7e1c1.

Reply:
We've discussed that change here: https://github.com/openwrt/openwrt/pull/15460#issuecomment-2108069634
I agree with @M95D.

Reply:
One more moment: default value is compile-time constant defined with `CONFIG_DROPBEAR_DEFAULT_RECV_WINDOW` which is `32768` for low-memory devices and `262144` otherwise. The one may change this value either via build config (compile-time) or via uci config (run-time). IMO it seems legit.